src/commands/utility/holidays.js

```diff
@@ -0,0 +1,160 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../')
+const moment = require('moment')
+const fetch = require('node-fetch')
+
+const countries = {
+  AD: 'Andorra',
+  AL: 'Albania',
+  AR: 'Argentina',
+  AT: 'Austria',
+  AU: 'Australia',
+  AX: 'Åland Islands',
+  BB: 'Barbados',
+  BE: 'Belgium',
+  BG: 'Bulgaria',
+  BJ: 'Benin',
+  BO: 'Bolivia',
+  BR: 'Brazil',
+  BS: 'Bahamas',
+  BW: 'Botswana',
+  BY: 'Belarus',
+  BZ: 'Belize',
+  CA: 'Canada',
+  CH: 'Switzerland',
+  CL: 'Chile',
+  CN: 'China',
+  CO: 'Colombia',
+  CR: 'Costa Rica',
+  CU: 'Cuba',
+  CY: 'Cyprus',
+  CZ: 'Czechia',
+  DE: 'Germany',
+  DK: 'Denmark',
+  DO: 'Dominican Republic',
+  EC: 'Ecuador',
+  EE: 'Estonia',
+  EG: 'Egypt',
+  ES: 'Spain',
+  FI: 'Finland',
+  FO: 'Faroe Islands',
+  FR: 'France',
+  GA: 'Gabon',
+  GB: 'United Kingdom',
+  GD: 'Grenada',
+  GL: 'Greenland',
+  GM: 'Gambia',
+  GR: 'Greece',
+  GT: 'Guatemala',
+  GY: 'Guyana',
+  HN: 'Honduras',
+  HR: 'Croatia',
+  HT: 'Haiti',
+  HU: 'Hungary',
+  ID: 'Indonesia',
+  IE: 'Ireland',
+  IM: 'Isle of Man',
+  IS: 'Iceland',
+  IT: 'Italy',
+  JE: 'Jersey',
+  JM: 'Jamaica',
+  JP: 'Japan',
+  LI: 'Liechtenstein',
+  LS: 'Lesotho',
+  LT: 'Lithuania',
+  LU: 'Luxembourg',
+  LV: 'Latvia',
+  MA: 'Morocco',
+  MC: 'Monaco',
+  MD: 'Moldova',
+  MG: 'Madagascar',
+  MK: 'Macedonia',
+  MN: 'Mongolia',
+  MT: 'Malta',
+  MX: 'Mexico',
+  MZ: 'Mozambique',
+  NA: 'Namibia',
+  NE: 'Niger',
+  NI: 'Nicaragua',
+  NL: 'Netherlands',
+  NO: 'Norway',
+  NZ: 'New Zealand',
+  PA: 'Panama',
+  PE: 'Peru',
+  PL: 'Poland',
+  PR: 'Puerto Rico',
+  PT: 'Portugal',
+  PY: 'Paraguay',
+  RO: 'Romania',
+  RS: 'Serbia',
+  RU: 'Russia',
+  SE: 'Sweden',
+  SI: 'Slovenia',
+  SJ: 'Svalbard and Jan Mayen',
+  SK: 'Slovakia',
+  SM: 'San Marino',
+  SR: 'Suriname',
+  SV: 'El Salvador',
+  TN: 'Tunisia',
+  TR: 'Turkey',
+  UA: 'Ukraine',
+  US: 'United States',
+  UY: 'Uruguay',
+  VA: 'Vatican City',
+  VE: 'Venezuela',
+  VN: 'Vietnam',
+  ZA: 'South Africa',
+  ZW: 'Zimbabwe'
+}
+
+module.exports = class Holidays extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'holidays',
+      category: 'utility',
+      parameters: [{
+        type: 'string',
+        whitelist: Object.keys(countries),
+        missingError: ({ t }) => {
+          return new SwitchbladeEmbed()
+            .setTitle(t('commands:holidays.invalidCountry'))
+            .setDescription(Object.entries(countries)
+              .map(([countryCode, countryName]) =>
+                `${countryCode} - ${countryName}`
+              ).join('\n')
+            )
+        }
+      }, {
+        type: 'number',
+        required: false,
+        min: 0
+      }]
+    })
+  }
+
+  async run ({ t, author, channel }, countryCode = '', year = moment().year()) {
+    const embed = new SwitchbladeEmbed(author)
```

Comment:
if its not much to ask, would be nice a start typing event, most commands have it

Reply:
What do you mean by start typing event? Is this that disclaimer message below the chat saying "Member X is typing..."? That's cool, I'll include it.

Reply:
Yup! We usually add it to commands that make web requests, cause they might take some extra time to complete. This way the user knows the bot is working, and didn't crash or something xD

Make sure to startTyping as soon as the command is ran, and stopTyping as soon as the answer is sent.

Reply:
Done!